app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt

```diff
@@ -21,6 +21,8 @@ import com.hedvig.app.util.getWithDotNotation
 import com.hedvig.app.util.plus
 import com.hedvig.app.util.safeLet
 import com.hedvig.app.util.toStringArray
+import java.util.Stack
```

Comment:
This change seems to happen frequently on your commits 🤔 maybe there's something strange in your IDE settings? 🤔

Reply:
Oh, will check it out